rpc/proto/mayastor.proto

```diff
@@ -146,6 +146,7 @@ message Nexus {
   // Path to device file for the volume (missing if not published).
   // Missing property and empty string are treated the same.
   string device_path = 5;
+  uint64 rebuilds = 6;
```

Comment:
all of them seem to have a comment, maybe do the same?

Reply:
done